t/129-ssl-socket.t

```diff
@@ -693,7 +693,7 @@ SSL reused session
     server_tokens off;
     resolver $TEST_NGINX_RESOLVER ipv6=off;
     lua_ssl_trusted_certificate ../html/trusted.crt;
-    lua_ssl_verify_depth 1;
+    lua_ssl_verify_depth 0;
```

Comment:
Hmm, why is this necessary for OpenSSL 1.1?

Reply:
@agentzh 
`SSL_CTX_set_verify_depth` in OpenSSL 1.1.0 verifies
one more level. Since Nginx doesn't change the default value for
ssl_verfiy_depth, I just update the test cases instead of updating the
C code.

Reply:
@spacewander I see. Thanks for your explanation!